src/home-page/HomePage.tsx

```diff
@@ -1,35 +1,39 @@
 import React from 'react';
-import { useHistory } from 'react-router-dom';
-
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 import { MainContainer } from '../layout/components/MainContainer';
 import { Header } from './components/Header';
 import { Description } from './components/Description';
 import { GridContainer } from './components/GridContainer';
-import { ProfessionCard } from './components/ProfessionCard';
+import { ProfessionCard } from '../shared/components/ProfessionCard';
 import { ITProfessions } from '../data/ITProfessions';
 
-export const HomePage: React.FC<{}> = () => {
-  const history = useHistory();
-  const handleClick = (route: string) => {
-    history.push(`/details/${route}`);
-  };
-  return (
-    <MainContainer>
-      <Header />
-      <Description />
-      <GridContainer>
-        {ITProfessions.map(({ title, description, color, icon, id, route }) => (
+const StyledLink = styled(Link)`
+  display: flex;
+  align-items: stretch;
+  text-decoration: none;
+  :active {
+    color: inherit;
+  }
+`;
+
+export const HomePage: React.FC<{}> = () => (
+  <MainContainer>
+    <Header />
+    <Description />
+    <GridContainer>
+      {ITProfessions.map(({ title, description, color, icon, id, route }) => (
+        <StyledLink to={`/details/${route}`} key={id}>
           <ProfessionCard
+            isClickable
             key={id}
             title={title}
             description={description}
             avatarSrc={icon}
             avatarBackground={color}
-            clickHandler={handleClick}
-            route={route}
           />
-        ))}
-      </GridContainer>
-    </MainContainer>
-  );
-};
+        </StyledLink>
```

Comment:
By using Link element from router we can use card as link - for example open in new tab with Ctrl+Click. icClickable props is essential for right styling clickable/non clickable version of ProfessionCard.